apps/backoffice/.env.development

```diff
@@ -32,6 +32,10 @@ AZURE_APIM=AZURE_APIM
 APIM_USER_GROUPS=apimessagewrite,apiinforead,apimessageread,apilimitedprofileread,apiservicewrite
 API_APIM_MOCKING=true
 
+AZURE_CREDENTIALS_SCOPE_URL=https://management.azure.com/.default
+AZURE_APIM_SUBSCRIPTIONS_API_BASE_URL=https://management.azure.com/subscriptions/
```

Comment:
chore: set this env vars also from terraform

Reply:
Added in 765804e5972245e00257df1fde092b26255bf3ca